### ISSUE TITLE
Mask R-CNN with ResNet-101-FPN-3x trimmed weights

### ISSUE THREAD

**Issue by duklin** [hpanwar08/detectron2#10]
Can you please provide trimmed version of the weights for the model mentioned in the title as there is trimmed version of the other model (Mask R-CNN with ResNeXt-101-32x8-FPN)

**Comment by hpanwar08** [comment]
You can extract the trimmed model from the full model. Full model contains model, optimizer, scheduler, iteration.
Use below code to get the trimmed model
```python
import torch
m = torch.load("model_final.pth")
trimmed = m["model"]
torch.save(trimmed, "trimmed_model.pth")
```
